website/versioned_docs/version-0.60/win10-compat.md

```diff
@@ -22,4 +23,6 @@ The following table captures the versions of Windows 10 that a React Native for
 
 | React Native Windows | Target OS Version | Minimum OS Version |
 | :--: | :-: | :-: |
-| 0.60 | **May 2019 update**<br> Version 1903 ; Build 10.0.18362.1 | **Creators Update**<br> Version 1703 ; Build 10.0.15063.468 |
+| 0.66+ | **May 2020 update**<br> Version 2004 ; Build 10.0.19041.0 | **Fall Creators Update**<br> Version 1709 ; Build 10.0.16299.91 |
```

Comment:
Does this need to be updated due to the changes for WinUI 2.8 bumping the minimum version in 0.72?

Reply:
Yes it will be! I'm going to have yajur add that change when he cuts the docs for 72.